backend/src/nodes/nodes/pytorch/inpaint.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
+
+import gc
+
+import numpy as np
+import torch
+
+from ...impl.pytorch.types import PyTorchInpaintModel
+from ...impl.pytorch.utils import (
+    np2tensor,
+    tensor2np,
+    to_pytorch_execution_options,
+)
+from ...node_base import NodeBase
+from ...node_factory import NodeFactory
+from ...properties import expression
+from ...properties.inputs import ImageInput
+from ...properties.inputs.pytorch_inputs import InpaintModelInput
+from ...properties.outputs import ImageOutput
+from ...utils.exec_options import ExecutionOptions, get_execution_options
+from . import category as PyTorchCategory
+
+
+@NodeFactory.register("chainner:pytorch:inpaint")
+class InpaintNode(NodeBase):
+    def __init__(self):
+        super().__init__()
+        self.description = "Inpaint an image with given mask."
+        self.inputs = [
+            ImageInput(channels=3),
+            ImageInput(label="Mask", channels=1),
+            InpaintModelInput(),
+        ]
+        self.outputs = [
+            ImageOutput(
+                image_type=expression.Image(
+                    width="Input0.width & Input1.width",
+                    height="Input0.height & Input1.height",
+                    channels="Input0.channels",
+                )
```

Comment:
We know that input0 always is RGB, so the output is also RGB. It's better to use the `channels` parameter of the `ImageOutput` in that case, since it will check the type at runtime.

```suggestion
                image_type=expression.Image(
                    width="Input0.width & Input1.width",
                    height="Input0.height & Input1.height",
                ),
                channel=3,
```

Reply:
lol I shouldn't have committed this suggestion directly. I didn't notice you had a typo. 😄I've been led astray!